lib/aur-srcver

```diff
@@ -47,10 +52,15 @@ trap 'trap_exit' EXIT
 cd "$tmp" || exit
 parallel --will-cite --nice 10 -j +2 --joblog makepkg_log --results '{#}_makepkg' \
   'cd {}; makepkg --skipinteg --noprepare -od' >/dev/null 2>&1 ::: "${arg_path[@]}"
-num_failed=$?
+exitcode=$?
+
+if ((exitcode > 101)); then
+    printf >&2 '%s: error running "parallel", exit code %d\n' "$argv0" "$exitcode"
+    exit $exitcode
```

Comment:
quoted in line above, but not here (doesn't really matter for integers, but might as well be uniform)